gstcefsrc.cc

```diff
@@ -228,17 +228,56 @@ class AudioHandler : public CefAudioHandler
     IMPLEMENT_REFCOUNTING(AudioHandler);
 };
 
+class DisplayHandler : public CefDisplayHandler {
+public:
+  DisplayHandler(GstCefSrc *element) : mElement(element) {}
+
+  ~DisplayHandler() = default;
+
+  virtual bool OnConsoleMessage(CefRefPtr<CefBrowser>, cef_log_severity_t level,
+                                const CefString &message, const CefString &source, int line) override {
+    GstDebugLevel gstLevel = GST_LEVEL_NONE;
+    switch (level) {
+    case LOGSEVERITY_DEFAULT:
+    case LOGSEVERITY_INFO:
+      gstLevel = GST_LEVEL_INFO;
+      break;
+    case LOGSEVERITY_DEBUG:
+      gstLevel = GST_LEVEL_DEBUG;
+      break;
+    case LOGSEVERITY_WARNING:
+      gstLevel = GST_LEVEL_WARNING;
+      break;
+    case LOGSEVERITY_ERROR:
+    case LOGSEVERITY_FATAL:
+      gstLevel = GST_LEVEL_ERROR;
+      break;
```

Comment:
This is for `console.error` right? Wouldn't it spam gst console? Not sure if I would put them in the same category

Reply:
What do you suggest then?

Reply:
I think they all should go under `GST_LEVEL_DEBUG` or `GST_LEVEL_INFO`. Depends on the use case I guess. In my case when I run my pipeline using third party website I wouldn't want to see their website logs in my gst output.

Maybe `GST_LEVEL_INFO` with a prop to turn it on or just `GST_LEVEL_DEBUG`.

What is your use case to have this? The website works fine in chrome and `cefsrc` have weird behavior?

Reply:
Yeah. I wonder, how about using a dedicated gst log category, like `cefconsole`? So that filtering might be easier, like `cefsrc:6,cefconsole:4` for instance.

Reply:
I think the separate category makes this fine, let's revisit if this proves too spammy :)